docs/source/rbac/roles.md

```diff
@@ -0,0 +1,60 @@
+# Roles
+JupyterHub provides three **default roles** which are automatically loaded to the database at the startup:
+
+```{admonition} **Default roles**
+- **_user_** role carries single `all` scope that grants _least user access_ to perform only default user actions.
+- **_admin_** role contains all available scopes and grants full rights to all actions similarly to the current admin status.
+- **_server_** role allows for posting activity only through the single `users:activity` scope.
+```
+
+Roles can also be customly defined and assigned to users, services, groups and tokens. 
+
+**_Users_** and **_services_** are assigned a default role if no custom role is requested based on their admin status.
+
+**_Tokens_**’ roles cannot grant the token higher permissions than their owner’s roles. If no specific role is requested, tokens are assigned the default _user_ role. 
+
+**_Groups_** do not require any role and are not assigned any roles by default. Once group roles are defined and assigned, the privileges of each group member are extended with the group roles in the background during the API request permission check. This is useful for requesting the same permissions for several users.
```

Comment:
> in the background

Could you explain what this means? Are a user's privileges the combination of their own role and their group's role, or does `background` imply something more complicated?

Reply:
The  text was rephrased to "..., the privileges of each group member are extended when needed (see ref for more details)" and reference to the relevant section added.